index.html

```diff
@@ -3602,13 +3634,24 @@ <h3>Parsing content</h3>
           </li>
           <li>
             Let |record:NDEFRecord| be the result of <a>parse an NDEF record</a>
-            on |ndef|.
+            given |ndef| and |context|.
           </li>
           <li>
             If |record| is not `null`, <a>append</a> |record| to |records|.
           </li>
           <li>
-            If |messageEnd| is `true`, abort these sub-steps.
+            If |messageEnd| is `true`,
+              <ol>
+                <li>
+                  If |context| is not `"smart-poster"`, abort these sub-steps.
+                </li>
+                <li>
+                  If |records| does not contain a <a>URI record</a>
+                  for which the protocol is supported by the UA, or if it
+                  contains more than one <a>type record</a>, <a>size record</a>
+                  or <a>action record</a>,
+                  discard all records in |records| and abort these sub-steps.
```

Comment:
How would we do if |records| contains some other records than URI record, title record, icon record, size record, type record, action record? e.g. an external type record.

Reply:
The NFC spec allows that.